src/schema/v2/types/bidder_position_result.ts

```diff
@@ -13,14 +13,15 @@ export const BidderPositionResultType = new GraphQLObjectType<
     },
     messageHeader: {
       type: GraphQLString,
-      resolve: ({ message_header }) => message_header,
     },
     messageDescriptionMD: {
       type: GraphQLString,
-      resolve: ({ message_description_md }) => message_description_md,
     },
     position: {
       type: BidderPosition.type,
     },
+    rawError: {
+      type: GraphQLString,
+    },
```

Comment:
Like; if these were errors I'd expect one of the following: this information is in the errors key not the normal returned result. Or we do the `FooOrError` type thing.